### PR TITLE
Support Python 2 and 3

### PR DESCRIPTION
**Pull request by pmclanahan**
First attempt at simultaneous py2 and py3 support.

### CODE REVIEW

File: product_details/storage.py [hunk 1]
```diff
@@ -173,14 +175,14 @@ def update(self, name, content, last_modified):
             log.debug('Writing new copy of %s to %s.' % (
                 name, self.json_dir))
             tf = tempfile.NamedTemporaryFile(delete=False)
-            tf.write(content)
+            tf.write(binary_type(content.encode('utf8')))
```
Comment (Osmose) [inline product_details/storage.py]: Is `binary_type` necessary here? From what I just read, in Python 3 encode returns a `bytes` object already, and in Python 2 it returns an 8-bit `str`.

File: tox.ini [hunk 2]
```diff
@@ -1,17 +1,21 @@
 [tox]
 args_are_paths = false
-envlist =
-    py27-{1.7,1.8}
+skip_missing_interpreters = true
+envlist = py{27,33,34,py}-dj{17,18},py35-dj18,flake8
```
Comment (Osmose) [inline tox.ini]: Now that it's been almost 2 months, you should add Django 1.9 here as well. :D

**Comment by Osmose** [comment]
r+, this looks great to me, and the tests all pass locally for me. Nice work!

File: tox.ini (hunk 2 above)
Comment (pmclanahan) [inline tox.ini]: Good point :)

File: product_details/storage.py (hunk 1 above)
Comment (pmclanahan) [inline product_details/storage.py]: Looks like you're right. Thanks.

File: tox.ini (hunk 2 above)
Comment (pmclanahan) [inline tox.ini]: Turns out this is more than just adding 1.9 to the tox file. I'll fix this in a subsequent PR.

**Comment by pmclanahan** [comment]
Updated some things from review, and found an error with how I was using urllib from six in the management command. I think all's well now.

Oh, and I updated the release.sh script to just build the sdist and wheel and recommend using `twine` to upload to PyPI.

**Pull request merged**